barcode-frontend-vue/README.md

```diff
@@ -1,4 +1,4 @@
-# template-frontend-vue
```

Comment:
Can you please remove the entire barcode-frontend-vue folder

Reply:
Addressed.